h2o-docs/src/booklets/v2_2015/source/sw/sections/api.tex

```diff
@@ -3,27 +3,18 @@ \section{Programming API}
 \subsection{Starting H2O Services}
 
 \begin{lstlisting}[style=Scala]
-val sc: SparkContext = ...
-val hc = H2OContext.getOrCreate(sc)
+import org.apache.spark.h2o._
+val hc = new H2OContext(getOrCreate(spark)}
 \end{lstlisting} 
-or:
-\begin{lstlisting}[style=Scala]
-val sc: SparkContext = ...
-val hc = new H2OContext(sc).start()
-\end{lstlisting}
 
 When the number of Spark nodes is known, it can be specified in the \texttt{getOrCreate} call:
 
 \begin{lstlisting}[style=Scala]
-val hc = H2OContext.getOrCreate(sc, numOfSparkNodes)
+val hc = H2OContext.getOrCreate(spark, numOfSparkNodes)
```

Comment:
`H2OContext.getOrCreate` no longer accept the second argument.

First argument is always `spark context` or `spark session` and the second one can be `H2OConf`.

So even this `When the number of Spark nodes is known, it can be specified ` can be deleted